statvar_imports/southkorea_statistics/demographics/population_pvmap.csv

```diff
@@ -0,0 +1,41 @@
+key,p1,v1,p2,v2,p3,v3,p4,v4
+"By administrative divisions(eup, myeon, dong)",observationAbout,{Data},measuredProperty,count,,,,
+Total population[명],value,{Number},populationType,Person,measuredProperty,count,,
+Male[명],value,{Number},populationType,Person,gender,Male,,
+Female[명],value,{Number},populationType,Person,gender,Female,,
+Korean - total[명],value,{Number},populationType,Person,nativity,Native,,
+Korean - male[명],value,{Number},populationType,Person,nativity,Native,gender,Male
+Korean - Female[명],value,{Number},populationType,Person,nativity,Native,gender,Female
+Foreigner-Total[명],value,{Number},populationType,Person,nativity,ForeignBorn,,
+Foreigner-Male[명],value,{Number},populationType,Person,nativity,ForeignBorn,gender,Male
+Foreigner-Female[명],value,{Number},populationType,Person,nativity,ForeignBorn,gender,Female
+Household-Total[가구],value,{Number},populationType,Household,,,,
+Type of occupancy[가구],#ignore,ignore,value,1,,,,
+Institutional households[가구],value,{Number},populationType,Household,householdType,InstitutionalHousehold,,
+Foreigner household[가구],value,{Number},populationType,Household,nativity,ForeignBorn,,
+Housing units-Total[호],value,{Number},populationType,HousingUnit,,,,
+Detached dwelling[호],#ignore,Not a high level variable1,value,1,,,,
+Apartment[호],value,{Number},populationType,HousingUnit,residenceType,Apartment,,
+Row house[호],value,{Number},populationType,HousingUnit,residenceType,RowHouse,,
+Apartment unit ina private house[호],#ignore,Not a high level variable2,value,1,,,,
+House within commercial building[호],#ignore,Not a high level variable3,value,1,,,,
+Living quaters other than housing unit[호],#ignore,Not a high level variable4,value,1,,,,
+2015 Year,observationDate,2015,,,,,,
```

Comment:
while mapping individual years works, it would be better to use  a regex or eval to extract the YYYY number from the column value so that the same pvmap can work for any future year beyond 2030 as well.

for eg:
`PERIOD,#Regex,(?P<observationDate>[0-9]{4}).*`